blacs/front_panel_settings.py

```diff
@@ -46,7 +46,10 @@ def setup(self,blacs):
         self.blacs = blacs
 
     def restore(self):
-        
+
+        # See if connection table exists in apparatus h5 file
+        # Return if not
+        #
```

Comment:
Could we move this comment to be adjacent to the code that actually implements it?